pool/test/session_test.ml

```diff
@@ -523,9 +527,13 @@ let cancel_no_reason () =
   in
   let res =
     SessionC.Cancel.(
-      [ "reason", [ "" ]; "email", [ "true" ]; "sms", [ "true" ] ]
+      [ "reason", [ "" ]; "notify_via", [ "email" ] ]
```

Comment:
Is the requirement to notify on a single channel, or do we need the possibility to notify on more than one at a time (e.g. email and SMS together)?

Reply:
That is a good question. I will check.